File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-ui/src/app/modules/user-management/user/dto/user-dto.ts
import { PageableDTO } from "../../../common-dto/common-dto";

export class UserFilterDTO extends PageableDTO {
    username!: string;
}

export class UserDTO {

    id!: number;
    username!: string;
    password!: string;
    fullName!: string;
    roles!: string[];
    secretAnswerOne!: string;
    secretResponseOne!: string;
    secretAnswerTwo!: string;
    secretResponseTwo!: string;

}

File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-api/src/main/java/com/cbi/ccr/csw/dashboard/user/UserDTO.java
package com.cbi.ccr.csw.dashboard.user;

import java.util.Set;

import javax.persistence.Column;

import com.cbi.ccr.csw.dashboard.user.dto.RoleName;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserDTO {
	
	private Long id;
	private String username;
	private String password;
	private String currentPassword;
	private String fullName;
	private String refreshToken;
	private Set<RoleName> roles; // comma separated
	private String secretAnswerOne;
	private String secretAnswerTwo;
	private String secretResponseOne;
	private String secretResponseTwo;

}
